srv/admin-service.cds

```diff
@@ -1,38 +1,18 @@
 using {sap.common.Languages as CommonLanguages} from '@sap/cds/common';
-using {my.bookshop as my} from '../db/index';
+using {my.bookshop as my} from '../db/schema';
 
-@path : 'admin'
-service AdminService @(requires : 'admin') {
-  entity Books   as projection on my.Books actions {
-    action addToOrder(order_ID : UUID, quantity : Integer) returns Orders;
+@path: 'admin'
+service AdminService @(requires: 'admin') {
+  @cds.search: {
+    descr,
+    title
   }
```

Comment:
Why was this annotation added?

Reply:
From here https://github.com/SAP-samples/cloud-cap-samples-java/blob/5371e0532fda61983a73267eea08bfbeb2da2627/srv/admin-service.cds#L27